qiskit/opflow/operator_globals.py

```diff
@@ -32,8 +33,12 @@
 # Immutable convenience objects
 
 
+@deprecate_func(
+    since="0.24.0",
+    additional_msg="For code migration guidelines, visit https://qisk.it/opflow_migration.",
+)
 def make_immutable(obj):
-    """Delete the __setattr__ property to make the object mostly immutable."""
+    r"""Deprecate\: Delete the __setattr__ property to make the object mostly immutable."""
```

Comment:
This method is not used outside or operator globals. It could have been private rather than seeming to be public. Either way the intent was that this function was more an internal function in opflow and it is not linked in the API ref. so the spelling without a `d` at the end will only ever be seen if you look at the source code - hence it could just as well be left as it.